website/src/app/kb/authenticate/okta/readme.mdx

```diff
@@ -0,0 +1,262 @@
+**Client secret**. Copy these values and paste them into the setup form in your
+Firezone admin portal.
+
+Optionally, enable the **Require PKCE as additional verification** setting for
+added security.
+
+<Link
+  href="/images/kb/authenticate/okta/8-client-credentials.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/8-client-credentials.png"
+    alt="Client credentials"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Scroll down to **Refresh Token** and ensure **Use persistent token** is
+selected.
+
+<Link
+  href="/images/kb/authenticate/okta/9-verify-refresh-token-behavior.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/9-verify-refresh-token-behavior.png"
+    alt="Verify refresh token behavior"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+### Step 4: Assign scopes and groups
+
+In the app integration settings in Okta, click **Assignments** and then the
+**Assign** button.
+
+<Link
+  href="/images/kb/authenticate/okta/10-assign-groups-people.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/10-assign-groups-people.png"
+    alt="Assign groups to people"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Ensure the `okta.groups.read` and `okta.users.read` scopes are granted.
+
+<Link
+  href="/images/kb/authenticate/okta/11-grant-groups-read-scope.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/11-grant-groups-read-scope.png"
+    alt="Grant groups read scope"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+<Link
+  href="/images/kb/authenticate/okta/12-grant-users-read-scope.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/12-grant-users-read-scope.png"
+    alt="Grant users read scope"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+<Link href="/images/kb/authenticate/okta/13-verify-scopes.png" target="_blank">
+  <Image
+    src="/images/kb/authenticate/okta/13-verify-scopes.png"
+    alt="Verify scopes"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+### Step 5: Complete the Okta provider setup in Firezone
+
+Go back to the **General** tab in the app integration settings in Okta and copy
+your **Okta account domain** shown in the top-right corner.
+
+Enter this value into the setup form in your Firezone admin portal.
+
+<Link
+  href="/images/kb/authenticate/okta/14-find-okta-account-domain.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/14-find-okta-account-domain.png"
+    alt="Find Okta account domain"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Ensure all fields are filled out, and click **Connect Identity Provider**.
+
+If you get successfully redirected back to your Firezone admin dashboard, you're
+done! Your Okta provider is now successfully configured. Users in your
+organization can now authenticate to Firezone using this provider and user /
+group sync will occur every few minutes.
```

Comment:
Same comment here as in the MS Entra ID page, just letting people know that users can login after the first directory sync.